OctoAwesome/OctoAwesome.Runtime/DiskPersistenceManager.cs

```diff
@@ -22,20 +22,20 @@ public class DiskPersistenceManager : IPersistenceManager
         private const string ColumnFilename = "column_{0}_{1}.dat";
 
         private DirectoryInfo root;
-        private ISettings Settings;
-
-        private IDefinitionManager definitionManager;
+        private ISettings Settings;        
         private IExtensionResolver extensionResolver;
-        private IResourceManager manager;
-
-
-
-        public DiskPersistenceManager(IExtensionResolver extensionResolver, IDefinitionManager definitionManager, IResourceManager manager, ISettings Settings)
+        private IDefinitionManager definitionmanager;
+        /// <summary>
+        /// Manager for persistance at lokal Disk.
+        /// </summary>
```

Comment:
Hast du die Doku-Kommentare gemacht? Wir hatten uns darauf geeinigt, dass diese in Deutsch auszuführen sind. Bitte keine Diskussion - hatten wir über das Thema schon genug!